aleph-client/src/balances.rs

```diff
@@ -0,0 +1,13 @@
+use crate::AnyConnection;
+use primitives::Balance;
+
+/// Reads from the storage how much balance is currently on chain.
+///
+/// Performs a single storage read.
+pub fn total_issuance<C: AnyConnection>(connection: &C) -> Balance {
+    connection
+        .as_connection()
+        .get_storage_value("Balances", "TotalIssuance", None)
+        .expect("Key `Balances::TotalIssuance` should be present in storage")
+        .unwrap_or(0)
```

Comment:
Here we change the semantics of this method, previously we `panic`ed when `TotalIssuance` cannot be decoded, now we return 0. Former seems to be more defensive but in light of this is in library, imho we want to be more strict just in case we slip a false positive e2e test run only if storage was borked (unlikely, I know) by somehow e2e passing with 0 value (depends on e2e test logic in general case, agree with currents tests we would catch error anyway ). One can even say this is a major semver change,

This can be fixed by passing some lambda what user of this function want to do in case entry cannot be decoded, ie wither `unwrap` anyway or return 0 etc

Another possibility is that we would go into direction that upon such error scenario we return the default value for given storage value

Reply:
I'm about to start implementing such task: https://cardinal-cryptography.atlassian.net/browse/A0-1096, where such concerns have even broader consequences. I propose leaving this particular method as it is (or I can revert the behavior - whatever you prefer) and then, in the next PR, I will provide two methods for reading storage (and analogous pair for constants):
 - `read_storage(connection: AnyConnection, module: String, key: String)` - panicable, force reading
 - `read_storage_or_else(connection: AnyConnection, module: String, key: String, handler: Fn: () -> T)` - defensive reading with possibility of providing default or some handling

wdyt?

Reply:
Great, let's revert this particular behavior if it's not a big problem (let's try to be consistent with this semver semantics). 

Those two methods sound like a decent design.